src/styles/WishlistScreen/listItems.jsx

```diff
@@ -0,0 +1,26 @@
+import { StyleSheet, StatusBar } from "react-native";
+
+export const sectionList = StyleSheet.create({
+    container: {
+      flex: 1,
+      paddingTop: StatusBar.currentHeight,
+      marginHorizontal: 16,
+    },
+    item: {
+      backgroundColor: '#f9c2ff',
```

Comment:
Please add this color to you library colors

Reply:
Done.